filebeat/docs/inputs/input-syslog.asciidoc

```diff
@@ -32,15 +33,20 @@ Example configurations:
 ----
 {beatname_lc}.inputs:
 - type: syslog
+  format: auto
   protocol.unix:
     path: "/path/to/syslog.sock"
 ----
 
 ==== Configuration options
 
-The `syslog` input supports protocol specific configuration options plus the
+The `syslog` input configuration includes format, protocol specific options, and the
 <<{beatname_lc}-input-{type}-common-options>> described later.
 
+===== `format`
+
+Which syslog variant to use: `rfc3164`, `rfc5424`, or `auto` to attempt to detect the variant automatically from the log entries. The default is `rfc3164`. `rfc5424` and `auto` options are currently in beta.
```

Comment:
```suggestion
The syslog variant to use when attempting to detect the variant automatically from the log entries. Valid values are `rfc3164`, `rfc5424`, or `auto`. The default is `rfc3164`. The `rfc5424` and `auto` options are currently in beta.
```

Reply:
Suggest moving the list of valid values later in case we add support for more variants. 3 is OK, 4 would be too much.

Reply:
The suggested edit doesn't quite work since auto-detection is only used for the `auto` setting -- I tried to clarify, let me know if this version works.